content/applications/finance/fiscal_localizations/united_arab_emirates.rst

```diff
@@ -0,0 +1,222 @@
+   Odoo; or are specific to the **U.A.E localization package**.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Code
+     - Account Name
+     - Type
+   * - 102011
+     - Accounts Receivable
+     - Receivable
+   * - 102012
+     - Accounts Receivable (PoS)
+     - Receivable
+   * - 201002
+     - Payables
+     - Payable
+   * - 101004
+     - Bank
+     - Bank and Cash
+   * - 105001
+     - Cash
+     - Bank and Cash
+   * - 100001
+     - Liquidity Transfer
+     - Current Assets
+   * - 101002
+     - Outstanding Receipts
+     - Current Assets
+   * - 101003
+     - Outstanding Payments
+     - Current Assets
+   * - 104041
+     - VAT Input
+     - Current Assets
+   * - 100103
+     - VAT Receivable
+     - Non-current Assets
+   * - 101001
+     - Bank Suspense Account
+     - Current Liabilities
+   * - 201017
+     - VAT Output
+     - Current Liabilities
+   * - 202001
+     - End of Service Provision
+     - Current Liabilities
+   * - 202003
+     - VAT Payable
+     - Non-current Liabilities
+   * - 999999
+     - Undistributed Profits/Losses
+     - Current Year Earnings
+   * - 400003
+     - Basic Salary
+     - Expenses
+   * - 400004
+     - Housing Allowance
+     - Expenses
+   * - 400005
+     - Transportation Allowance
+     - Expenses
+   * - 400008
+     - End of Service Indemnity
+     - Expenses
+
+Taxes
+=====
+
+To access your taxes, head to :menuselection:`Accounting --> Configuration --> Taxes`.
+Activate/deactivate, or :doc:`configure </applications/finance/accounting/taxation/taxes/taxes>` the
+taxes relevant to your business by clicking on them. Remember only to set accounts on the **5%** tax
+group, as other groups do not need closing.
+
+.. tip::
+   Use the :guilabel:`Group By --> Tax Type` filter to group taxes by type and have a clear overview
+   of the different taxes.
+
+.. image:: united_arab_emirates/uae-localization-taxes.png
+   :align: center
+   :alt: Preview of the U.A.E. localization package's taxes.
+
+Tax report
+----------
+
+To create a tax report when closing your period, head to :menuselection:`Accounting --> Reporting
+--> Tax Report`. The closing period corresponds to the period configured in
+:menuselection:`Accounting --> Configuration --> Settings --> Taxes`, but you can always change it
+by clicking on the **calendar icon** and selecting the period you wish to close.
+
+Journals
+--------
+
+To configure your journals, go to :menuselection:`Accounting --> Configuration --> Journals`. You
+can modify the pre-configured journals according to your needs or create new journals with five
+possible types: **Sales, Purchase, Bank, Cash**, and **Miscellaneous**.
+
+Currency exchange rates
+-----------------------
+
+Currency exchange rates are automatically updated using the U.A.E Central Bank rates, but by default
+the interval is set to *manual*. If you wish to use the automated currency rate update, change the
```

Comment:
I would put the word "interval" in guilabel to highlight the fact that it's the name of the field here?